dis_snek/client/const.py

```diff
@@ -173,5 +173,22 @@ class MentionPrefix(Sentinel):
     },
 )
 
+
+GUILD_WELCOME_MESSAGES = [
```

Comment:
Unsure how i feel about this being in `const`. While its technically correct, this is a constant, it feels like the wrong place. 

Note: This should not block merge, simply a discussion point

Reply:
To be honest same, but I saw the other guild thing in there. At first I was looking for somewhere to maybe store it in a discord const instead of client but, I'm not completely sure.